notebooks/qg/tableqg_inference.ipynb

```diff
@@ -0,0 +1,193 @@
+   "source": [
+    "<br>\n",
+    "\n",
+    "## Sample table\n",
+    "\n",
+    "Tables should be passed a `list` of `dicts`. Each `dict` will correspond to a table with keys `\"header\"` and `\"rows\"`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "18419939",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "+---------------+-------+---------------+----------------+--------------------+----------------+\n",
+      "| Player        |   No. | Nationality   | Position       | Years in Toronto   | School Team    |\n",
+      "+===============+=======+===============+================+====================+================+\n",
+      "| Antonio Lang  |    21 | United States | Guard-Forward  | 1999-2000          | Duke           |\n",
+      "+---------------+-------+---------------+----------------+--------------------+----------------+\n",
+      "| Voshon Lenard |     2 | United States | Guard          | 2002-03            | Minnesota      |\n",
+      "+---------------+-------+---------------+----------------+--------------------+----------------+\n",
+      "| Martin Lewis  |    32 | United States | Guard-Forward  | 1996-97            | Butler CC (KS) |\n",
+      "+---------------+-------+---------------+----------------+--------------------+----------------+\n",
+      "| Brad Lohaus   |    33 | United States | Forward-Center | 1996               | Iowa           |\n",
+      "+---------------+-------+---------------+----------------+--------------------+----------------+\n",
+      "| Art Long      |    42 | United States | Forward-Center | 2002-03            | Cincinnati     |\n",
+      "+---------------+-------+---------------+----------------+--------------------+----------------+\n"
+     ]
+    }
+   ],
+   "source": [
+    "table_list = [\n",
+    "    {\"header\": [\"Player\", \"No.\", \"Nationality\", \"Position\", \"Years in Toronto\", \"School Team\"],\n",
+    "      \"rows\": [\n",
+    "            [\"Antonio Lang\", 21, \"United States\", \"Guard-Forward\", \"1999-2000\", \"Duke\"],\n",
+    "            [\"Voshon Lenard\", 2, \"United States\", \"Guard\", \"2002-03\", \"Minnesota\"],\n",
+    "            [\"Martin Lewis\", 32, \"United States\", \"Guard-Forward\", \"1996-97\", \"Butler CC (KS)\"],\n",
+    "            [\"Brad Lohaus\", 33, \"United States\", \"Forward-Center\", \"1996\", \"Iowa\"],\n",
+    "            [\"Art Long\", 42, \"United States\", \"Forward-Center\", \"2002-03\", \"Cincinnati\"]\n",
+    "        ]\n",
+    "    }\n",
+    "]\n",
+    "\n",
+    "print(tabulate(table_list[0]['rows'], headers=table_list[0]['header'], tablefmt='grid'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b002086c",
+   "metadata": {},
+   "source": [
+    "## Generate questions\n",
+    "\n",
+    "There are some arguments to control the type of questions generated.\n",
+    "#### Controls:\n",
```

Comment:
Look how this is done in tydiqa.ipynb perhaps you can do it similarly? The arguments are assigned as variables before being used with each have an explanation as a comment.

Reply:
This is closer to the haystack style, where the aim is to show how to reuse the functionality using few lines of code. Let us keep it this way.